stdlib/messaging/rabbitmq/src/main/ballerina/rabbitmq/channel.bal

```diff
@@ -92,15 +92,30 @@ public type Channel client object {
     # + queueName - The name of the queue.
     # + return - An error if an I/O error is encountered or nil if successful.
     public remote function queuePurge(string queueName) returns error? = external;
+
+    # Closes the RabbitMQ `Channel`.
+    #
+    # + closeCode - The close code (See under "Reply Codes" in the AMQP specification).
```

Comment:
shall we mention the AMQP version or provide a link to the spec?

Reply:
Added